sdk/tables/data-tables/src/TableClient.ts

```diff
@@ -473,6 +486,19 @@ export class TableClient {
     }
   }
 
+  /**
+   * Creates a new Batch to collect sub-operations that can be submitted together via submitBatch
+   * @param tableName The name of the table.
+   * @param entity The properties for the table entity.
+   * @param mode The different modes for updating the entity:
+   *             - Merge: Updates an entity by updating the entity's properties without replacing the existing entity.
+   *             - Replace: Updates an existing entity by replacing the entire entity.
+   * @param options The options parameters.
+   */
+  public createBatch(partitionKey: string): TablesBatch {
+    return createBatch(this.url, this.tableName, partitionKey, this.credential);
```

Comment:
the naming gets a little confusing, having a class method named the same thing as your helper method.